src/app/sidebar/components/Card.tsx

```diff
@@ -39,8 +77,8 @@ export function Card({ subject, title, code, selected }: PropsWithChildren<CardP
       my={1}
       cursor="pointer"
```

Comment:
For scheduler the cursor shouldn't be a pointer since clicking the card and not the buttons doesn't do anything